templates/components/datatable.html.twig

```diff
@@ -212,7 +212,7 @@
                                                    {% endif %}
                                                 </span>
                                         {% else %}
-                                            {{ entry[colkey] }}
+                                            {{ entry[colkey]|raw }}
```

Comment:
There is already a `raw_html` filter for that.

Reply:
if I change the formatting to ```raw_html``` I lose the ability to filter with a dropdown list (only a text field is provided)

![image](https://github.com/glpi-project/glpi/assets/7335054/3761a268-d6bf-452c-a2b3-f56f129c1ea0)

Reply:
Maybe add a new optional filter_formatters parameter that can be set for this template to specify the formatting. If none is specified for a column, use the formatting based on the column.

Example:
{% set filter_format = filter_formatters[colkey]|default(formatters[colkey]) %}

Reply:
It's simpler that you think.
@cedric-anne said there is already a formatter making the `|raw` filter above the else.

Check https://github.com/glpi-project/glpi/pull/16465/files/6d9a1413e6512a3cf4f89bb5fac8ca981ac646e0#diff-c43a216f1d73a092b92980d1fc0766a39e50ae64b5180e51d35aed3dc1a5d384L199-R200